components/dot_menu/dot_menu.tsx

```diff
@@ -1,13 +1,13 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
A contributor is working on migrating this file here: https://github.com/mattermost/mattermost-webapp/pull/6825
Not sure what we should do, but assuming all is alright with that PR we should probably incorporate his changes.

Reply:
thanks for tagging that other PR @devinbinnie . I have just now fixed all broken tests, etc and merged from the `master` branch. All checks should pass now - if you can review and fast-track #6825 - then this PR might be cleaner.